storage/rwtree_test.go

```diff
@@ -13,14 +13,15 @@ import (
 func TestSave(t *testing.T) {
 	db := dbm.NewMemDB()
 	rwt := NewRWTree(db, 100)
-	foo := bz("foo")
-	gaa := bz("gaa")
-	dam := bz("dam")
+	foo := []byte("foo")
+	gaa := []byte("gaa")
+	dam := []byte("dam")
 	rwt.Set(foo, gaa)
-	rwt.Save()
+	_, _, err := rwt.Save()
+	require.NoError(t, err)
 	assert.Equal(t, gaa, rwt.Get(foo))
 	rwt.Set(foo, dam)
-	_, _, err := rwt.Save()
+	_, _, err = rwt.Save()
```

Comment:
Do you think it would be worth adding a check that the version has been incremented?